authors/apps/articles/tests/test_create_article.py

```diff
@@ -0,0 +1,199 @@
+                         response.data['description'])
+
+    def test_unauthorized_article_update(self):
+        """Test to update an article created by another user"""
+        token = self.signup_user_one()
+        del self.create_article_data['images']
+        self.client.post(self.article_listcreate,
+                         self.create_article_data,
+                         format='json',
+                         HTTP_AUTHORIZATION='token {}'.format(token))
+        token2 = self.signup_user_two()
+        response = self.client.put(reverse('articles:articles-retrieveupdate',
+                                   kwargs={'slug': 'a-new-story'}),
+                                   self.create_article_data2,
+                                   format='json',
+                                   HTTP_AUTHORIZATION='token {}'.format(token2))
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_authorized_article_update(self):
+        """Test to legitimately update an article created by a user"""
+        token = self.signup_user_one()
+        del self.create_article_data['images']
+        self.client.post(self.article_listcreate,
+                         self.create_article_data,
+                         format='json',
+                         HTTP_AUTHORIZATION='token {}'.format(token))
+        del self.create_article_data2['images']
+        response = self.client.put(reverse('articles:articles-retrieveupdate',
+                                   kwargs={'slug': 'a-new-story'}),
+                                   self.create_article_data2,
+                                   format='json',
+                                   HTTP_AUTHORIZATION='token {}'.format(token))
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(self.create_article_data2['title'],
+                         response.data['title'])
+        self.assertEqual(self.create_article_data2['body'],
+                         response.data['body'])
+        self.assertEqual(self.create_article_data2['description'],
+                         response.data['description'])
+
+    def test_to_delete_article(self):
+        """Test to delete an article"""
+        token = self.signup_user_one()
+        del self.create_article_data['images']
+        self.client.post(self.article_listcreate,
+                         self.create_article_data,
+                         format='json',
+                         HTTP_AUTHORIZATION='token {}'.format(token))
+        response = self.client.put(reverse('articles:articles-delete',
+                                   kwargs={'slug': 'a-new-story'}),
+                                   self.delete_article,
+                                   format='json',
+                                   HTTP_AUTHORIZATION='token {}'.format(token))
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
assert that the `is_deleted` field of the article is now true

Reply:
Done